### PR TITLE
Skip malware-detection tests on RHEL6/python2.6 (not supported)

### PR DESCRIPTION
**Pull request by mhuth**
* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

This is to correct problems encountered when running the malware-detection tests on RHEL6 systems outside the automated PR check environment. Malware detection won't be supported on RHEL6 so no need to check it works in that environment anyway.

### CODE REVIEW

**Review by bfahr — APPROVED** [review]
This looks good to me. @Glutexo I'll wait until you review before merging.

**Comment by Glutexo** [comment]
Although Python 2.6 / RHEL 6 is not supported, adding support here is very easy. But the original problem with the tests is that it uses an external utility and is dependent on an actual state of the machine.

**Pull request merged**
